website/pages/en/help.js

```diff
@@ -27,7 +27,12 @@ class Help extends React.Component {
       },
       {
         content:
-          'Many developers and users idle on Discord in [#relay](https://discord.gg/0ZcbPKXt5bX40xsQ) on [Reactiflux](https://www.reactiflux.com/).',
+          'Many developers and users idle on Slack in the [#relay](https://graphql.slack.com/messages/relay) channel of [the GraphQL Slack community](https://graphql-slack.herokuapp.com/).',
+        title: 'Slack',
```

Comment:
I remember I installed Discord just to go to #relay in Reactflux. 
* Are there more people at #relay on Slack or Discord? _from my experience Discord is too much quiet_
* Wouldn't it be better if we all stay in the same service?

Reply:
There are more people on graphql #relay